crates/speedy-transform/src/web_transform/remove_call.rs

```diff
@@ -9,30 +12,41 @@ struct RmCallStmt {
 
 impl VisitMut for RmCallStmt {
   fn visit_mut_block_stmt(&mut self, n: &mut BlockStmt) {
-    let mut rm_idx = vec![];
+    let mut rm_idx_set = HashSet::new();
     for (idx, stmt) in n.stmts.iter().enumerate() {
       if let Some(Some(call_expr)) = stmt.as_expr().map(|expr_stmt| expr_stmt.expr.as_call()) {
-        if let Some(callee) = call_expr.callee.as_expr() {
+        call_expr.callee.as_expr().and_then(|callee| {
           let callee_name = match callee.as_ref() {
-            Expr::Member(member) => member
-              .prop
-              .as_ident()
-              .map_or("".to_string(), |ident| ident.sym.to_string()),
```

Comment:
this `sym.to_string` will introduce an extra allocation. Instead, we could use `sym.clone()`, which is just a string interning clone, that is much cheaper than a `sym.to_string()`